src/utils/resolveLatestVersion.js

```diff
@@ -0,0 +1,26 @@
+/** @flow */
+/**
+ * Retrieve latest bit version from latest to number
+ * @param {Array<BitIds>} arr
+ * @param {string} id
+ * @returns {string}
+ */
+import maxBy from 'lodash.maxby';
+import { BitId } from '../bit-id';
+
+export default function getLatestVersionNumber(bitIds: BitId[] | string[], bitId: string | BitId) {
```

Comment:
Call this function from BitIds class, instead what we have there now. 
(And make sure the tests still pass)